functional/fixtures/units/global@.service

```diff
@@ -0,0 +1,5 @@
+[Service]
```

Comment:
Not sure whether it is a good idea. When you start 3 units in 3-machines cluster, fleet with schedule 3*3 units. Is it necessary in current test case?

```sh
core@coreos1 ~ $ fleetctl start gtemplate@{1..3}.service
Unit gtemplate@1.service 
Unit gtemplate@2.service 
Unit gtemplate@3.service 
Triggered global unit gtemplate@1.service start
Triggered global unit gtemplate@2.service start
Triggered global unit gtemplate@3.service start
core@coreos1 ~ $ fleetctl list-units
UNIT                    MACHINE                 ACTIVE  SUB
gtemplate@1.service     11594cc1.../coreos2     active  running
gtemplate@1.service     5b93e155.../coreos3     active  running
gtemplate@1.service     f6eaabe9.../coreos1     active  running
gtemplate@2.service     11594cc1.../coreos2     active  running
gtemplate@2.service     5b93e155.../coreos3     active  running
gtemplate@2.service     f6eaabe9.../coreos1     active  running
gtemplate@3.service     11594cc1.../coreos2     active  running
gtemplate@3.service     5b93e155.../coreos3     active  running
gtemplate@3.service     f6eaabe9.../coreos1     active  running
```